src/rabbit_stomp_processor.erl

```diff
@@ -589,45 +589,57 @@ do_subscribe(Destination, DestHdr, Frame,
                 rabbit_stomp_util:consumer_tag(Frame),
             case Prefetch of
                 undefined -> ok;
-                _         -> amqp_channel:call(
-                               Channel, #'basic.qos'{prefetch_count = Prefetch})
+                _         -> amqp_channel:call(Channel,
+                                               #'basic.qos'{
+                                                  prefetch_count = Prefetch
+                                                 })
```

Comment:
This change is totally unrelated to the pull request. If I'm reviewing the PR I need to understand why this has been changed. If somebody later is doing git-blame/history to understand some changes, they will find this and it will also be confusing